src/ide/completion/path.rs

```diff
@@ -98,12 +100,33 @@ pub fn path_suffix_completions(
             Some(CompletionItem {
                 label: last_segment.to_string(),
                 kind: Some(importable_completion_kind(*importable)),
+                label_details: Some(CompletionItemLabelDetails {
+                    detail: None,
+                    description: Some(path_str.to_string()),
+                }),
                 additional_text_edits,
                 ..CompletionItem::default()
             })
         })
         .unique_by(|completion| CompletionItemHashable(completion.clone()))
-        .collect()
+        .collect();
+
+    // Remove path label_details from all completions, that are NOT duplicated.
+    let completions_vec: Vec<_> = completions.clone();
+    let duplicate_labels: HashSet<_> = completions_vec
+        .iter()
+        .map(|item| &item.label)
+        .filter(|label| completions_vec.iter().filter(|item| &item.label == *label).count() > 1)
+        .cloned()
+        .collect();
+
+    for completion in &mut completions {
+        if !duplicate_labels.contains(&completion.label) {
+            completion.label_details = None;
+        }
+    }
```

Comment:
Please consider using `Itertools::counts` instead of counting manually:
```suggestion
    let label_counts = completions.iter().map(|item| item.label.clone()).counts();
    for completion in &mut completions {
        if label_counts[&completion.label] == 1 {
            completion.label_details = None;
        }
    }
```